tests/ts/lsfd/lsfd-functions.bash

```diff
@@ -83,3 +83,13 @@ lsfd_make_state_connected()
     # to that on newer kernels.
     sed -e 's/state=unconnected/state=connected/'
 }
+
+function lsfd_skip_factory
```

Comment:
`lsfd_check_factory` would be more in line with the other functions and clearer.

Reply:
Thank you. `lsfd_check_mkfds_factory` was the name I chose.